src/blankslate/index.scss

```diff
@@ -1 +1,2 @@
-@import '@primer/view-components/app/components/primer/beta/blankslate';
+@import '../support/index.scss';
+@import './blankslate.scss';
```

Comment:
I wanted to put this import back after https://github.com/primer/css/pull/2318 in case anyone is importing the `src/blankslate/blankslate.scss` file directly so it's not breaking.